__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -0,0 +1,1665 @@
+import { precalculationHelpers } from './helpers/wallet-precalculation.helper';
+import { GenesisWalletHelper } from './helpers/genesis-wallet.helper';
+import { delay, getRandomInt } from './utils/core.util';
+import {
+  createTokenHelper,
+  generateConnection,
+  generateWalletHelper,
+  stopAllWallets,
+  waitForTxReceived,
+  waitForWalletReady,
+  waitUntilNextTimestamp
+} from './helpers/wallet.helper';
+import HathorWallet from '../../src/new/wallet';
+import { HATHOR_TOKEN_CONFIG, TOKEN_MELT_MASK, TOKEN_MINT_MASK } from '../../src/constants';
+import transaction from '../../src/transaction';
+import { TOKEN_DATA } from './configuration/test-constants';
+import wallet from '../../src/wallet';
+import dateFormatter from '../../src/date';
+import { loggers } from './utils/logger.util';
+
+const fakeTokenUid = '000002490ab7fc302e076f7aab8b20c35fed81fd1131a955aebbd3cb76e48fb0';
+const sampleNftAddress = 'ipfs://bafybeiccfclkdtucu6y4yc5cpr6y3yuinr67svmii46v5cfcrkp47ihehy/albums/QXBvbGxvIDEwIE1hZ2F6aW5lIDI3L04=/21716695748_7390815218_o.jpg';
+
+describe('start', () => {
+  it('should start a wallet with no history', async () => {
+    const walletData = precalculationHelpers.test.getPrecalculatedWallet();
```

Comment:
I felt that using a helper to make almost 100% of a test would end up in me testing the _helper_ instead of the function I'm interested in.

So I preferred having some code lines duplicated on this point. Do you agree that's a good enough reason to not use the `generateWalletHelper`?

Reply:
If the method you are using uses the same method you want to test, what's the difference?

Reply:
I agree with Tulio and I feel that the tests should not use helpers if they are testing the method the helper uses since the helper adds another untested layer of code between the method being tested and the test.